app/views/pages/show.slim

```diff
@@ -1,4 +1,7 @@
 - meta optimization_tags: @page.meta_tags
+
+/Twitter Meta Tags for proper behavior there
+- meta twitter: {card: 'summary_large_image', domain: 'http://sumofus.org', site: @page.meta_twitter_handle, creator: @page.meta_twitter_handle, title: @page.title, description: truncate(@page.content, length: 140), img: {src: @page.primary_image.try(:content).try(:url)}}
```

Comment:
Oh, that's an interesting idea. It would obviate the need for testing and make sense to just move it to the view. I'll poke at that and see if it works.